### ISSUE TITLE
Add support for .ly

### ISSUE THREAD

**Issue by ravisorg** [glensc/nagios-plugin-check_domain#37]
```
Registrant:
   Google Inc.
   DNS Admin
   1600 Amphitheatre Parkway
   Mountain View
   CA
   United States of America
   Zip/Postal code: 94043
   Phone: +16503300100
   dns-admin@google.com

Domain Name:   google.ly

Administrative Contact:
   Google Inc.
   DNS Admin
   1600 Amphitheatre Parkway
   Mountain View
   CA
   US
   Zip/Postal code: 94043
   Phone: +16503300100
   dns-admin@google.com

Technical Contact:
   MarkMonitor
   Domain Admin
   10400 Overland Rd\\\\r\\\\nPMB 155
   Boise
   ID
   US
   Zip/Postal code: 83709
   Phone: +1.2083895740
   Fax: +1.2083895799
   ccops@markmonitor.com

Billing Contact:
   MarkMonitor
   Domain Administrator
   10400 Overland Rd\\\\r\\\\nPMB 155
   Boise
   ID
   US
   Zip/Postal code: 83709
   Phone: +1.2083895740
   Fax: +1.2083895799
   ccops@markmonitor.com

Created: 2007-10-03 13:36:48
Updated: 2014-09-05 18:17:29
Expired: 2015-10-03 13:36:48

Domain servers in listed order:
   ns4.google.com
   ns3.google.com
   ns2.google.com
   ns1.google.com

Domain Status: ACTIVE
--
Whois information provided by:
LY Registry
whois.nic.ly
-For Whois usage policy please check:
 http://whois.nic.ly/policy.php
```

**Comment by ravisorg** [comment]
Sorry, I don't mean to be demanding with these issue requests - I'm planning on taking a crack at them, so please don't take them as "DO THIS NOW, DAMMIT!" :)

**Comment by Elyrith** [comment]
This has the same "YYYY-MM-DD hh:mm:ss" format as both domains in issue #31, but those both start with "Expiration Time: ".

This is the first time I've built one of these, but I think this is correct.

```
# 2015-10-03 13:36:48
DATE_YYYY_MON_DD_HH_MM_SS = DATE_YYYY_MM_DD_DASH " " HH_MM_DD
```

**Comment by glensc** [comment]
merged via #40, thanks @Elyrith!

**Comment by ravisorg** [comment]
Wow, that was insanely fast! Thank you @Elyrith and @glensc! :D

**Comment by Elyrith** [comment]
No problem. These ones were easy.
#36 is going to be tricky for me, though.

**Comment by ravisorg** [comment]
@Elyrith Just saw your comment, hopefully you haven't spent much time on it yet because I just committed a fix for #36 :)
